openvinotoolkit/openvino
# Avgpool revise

1. Update the spec
2. Create unit-tests
3. Remove second constructor and add default values to the first one
4. Add type_prop tests for default values
5. Add 5D input single layer test instances
6. Update validate_and_infer_types method
7. Update infer_batched_pooling_forward and try_apply_auto_padding methods
8. Update auto_padding_spatial_dims_dynamic type_prop test for binary_conv, conv, deformable_conv, group_conv and max_pool

Changes in infer_batched_pooling_forward and try_apply_auto_padding required me to change the expected pads_begin and pads_end in 1 type_prop test for ops listed in point 8.

ticket: 37422
